.env.example

```
@@ -1,55 +1,263 @@
+# DB_BACKUP__STORAGE_URL
+# Purpose: S3 destination where db-backup uploads the dump (mapped to STORAGE__URL)
+# Format: s3://bucket/object-prefix
+# Example: s3://my-sms-gateway-backups/prod/
 DB_BACKUP__STORAGE_URL=s3://db-backup/prefix
+
+# DB_BACKUP__SCHEDULE
+# Purpose: Cron expression for the swarm-cronjob triggers db-backup (label swarm.cronjob.schedule)
+# Format: cron expression (6 fields); single quotes required in .env
+# Default: '@daily'
+# Example: '0 2 * * *'
 DB_BACKUP__SCHEDULE='@daily'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)